packages/theme/css/palette/neutral.css

```diff
@@ -5,12 +5,12 @@
   --salt-palette-neutral-primary-foreground: var(--salt-color-gray-900);
   --salt-palette-neutral-primary-foreground-disabled: var(--salt-color-gray-900-fade-foreground);
   --salt-palette-neutral-primary-separator: var(--salt-color-black-fade-separatorOpacity-primary);
-  --salt-palette-neutral-primary-border: var(--salt-color-gray-60);
+  --salt-palette-neutral-primary-border: var(--salt-color-gray-40);
   --salt-palette-neutral-primary-border-disabled: var(--salt-color-gray-60-fade-border);
```

Comment:
Are the other tokens not changing? cc @shey-v

Reply:
Yes, they will be an alpha of 40%.

Reply:
updated